runtime/src/lib.rs

```diff
@@ -318,8 +318,7 @@ impl pallet_babe::Config for Runtime {
 	type EquivocationReportSystem =
 		pallet_babe::EquivocationReportSystem<Self, Offences, Historical, ReportLongevity>;
 	type WeightInfo = ();
-	type MaxAuthorities = ConstU32<128>; // Q: maximum authority set size?
-									 //TODO: ^^^ actually consider what to set this to, I just picked a number
+	type MaxAuthorities = ConstU32<128>;
```

Comment:
Same question as GRANDPA  authorities. Do all active validators participate in Block Assignment?

Reply:
I don't think so. Substrate had this set to `10` in their test code. Assuming they are using the same test validator setup as grandpa I would expect this number to be a lot higher. The error didn't come from this pallet though so I think we can leave the number as is and cleaning up the comments is good enough for now

Reply:
this is smarter will add now